cloud-sql/mysql/client-side-encryption/snippets/cloud_kms_env_aead.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
It looks like there isn't an accompanying test for this - please add something that checks each of the operations here - if these are LROs and it isn't practical to wait for their entire completion (for example, training some ML models can take hours and we don't wait for their completion for the tests - we have a workaround) lmk and we can figure out a practical way to test